MultiSnap_timelapse_ef_230209_3.bsh

```diff
@@ -203,10 +200,12 @@ for(int qy=0;qy<panelnumy;qy++){
 			mm.core().sleep(intervalMs);
 
 			// ************ Chanell *********************
-			for(int j=0;j<channels.length;j++){
+			for(int j=0;j<channelsLength;j++){
```

Comment:
I found you defined channelsLength as 2 this time. I think **maybe** you did this as the channels.length in the last ver was 1 and it cause channel exchange failure. It may be caused by the `//String [] channels = {"CSUX 561"};` at 51. now since you comment it out, I think the channel.length will work well as before.

Reply:
I would like to disscus in WR. Thank you.